src/pages/MarketPage/AddonDetails/AddonDetails.jsx

```diff
@@ -106,14 +106,14 @@ const AddonDetails = ({ addon = {}, isLoading, onInstall, isUpdatingAll }) => {
     actionButton = (
       <Button
         variant="filled"
-        icon={'upgrade'}
+        icon={'download_for_offline'}
```

Comment:
relates to what @Innders wrote here, hope I understood it correctly https://discord.com/channels/517362899170230292/1240288129001328640/1240959874460811375

Reply:
<img width="839" alt="image" src="https://github.com/ynput/ayon-frontend/assets/16327908/126dea2c-9bb0-4e74-9203-a129cbc9d265">

Reply:
No i didnt, there should always be download.